### PR TITLE
scroll api

### PR DESCRIPTION
**Pull request by shlomialfasi**

### CODE REVIEW

File: scroll/readme.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+## General
+The Scroll API allows Logz.io users to run queries on the data in their account via REST (i.e. HTTP). The query language used is Elasticsearch Search API DSL, with certain limitations. We highly recommend you read the restrictions listed below (In the Request Body section) before beginning to use the API
+
+## License 
+Using the API requires a special license from Logz.io, and an API token which can be generated here: https://app.logz.io/#/dashboard/settings/shared-tokens 
+
+The number of queries executed is controlled and limited by Logz.io.
+
+The number of results are limited to 1,000.
```
Comment (segalziv) [inline scroll/readme.md:11]: it is limited to 1000 per a single page response. The number of results returned is limited by the account's plan and calculated on a daily basis.

File: scroll/readme.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+* "X-USER-TOKEN" -  The value should be the API token you generated (as explained in the License section above)
+* "Content-Type" - "application/json"  (required)
+
+### Request Body 
+Elasticsearch Search API Body as documented  in [Elaticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/search.html), with the following restrictions:
+* We support the following top-level elements in the Elasticsearch Search API:  `query`, `from`, `size`, `sort`, `_source`, `post_filter`, `scroll`, `scroll_id`.
+* When using the `query_string` element, you are not allowed to set its field named `allow_leading_wildcard` to true
+* When using the `wildcard` element, you are not allowed to have its value start with `*` or `?`
+* When using the `scroll_id` element, you are not allowed to use `query` element.
+* Your query will only run on data you sent today and yesterday. You can still specify a filter on the timestamp field to allow smaller time frame.
```
Comment (segalziv) [inline scroll/readme.md:30]: I don't think it is updated (in the previous search API as well). Need to verify, but I think that the query will run on today and yesterday by default, but you can still contact support and ask to extend it if needed

File: scroll/readme.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+* `scroll` element supports the following time units seconds/minutes/hours/days from [Time-Unit] (√https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#time-units).
```
Comment (segalziv) [inline scroll/readme.md:31]: there's an extra `√` in the url. please remove

File: scroll/readme.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+URL: https://api.logz.io/v1/scroll (US Accounts)
+     https://api-eu.logz.io/v1/scroll (EU Accounts)
+* Make sure to use https and not http, otherwise you will get a '403 Forbidden' response..
+
+### HTTP Headers
+* "X-USER-TOKEN" -  The value should be the API token you generated (as explained in the License section above)
+* "Content-Type" - "application/json"  (required)
+
+### Request Body 
+Elasticsearch Search API Body as documented  in [Elaticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/search.html), with the following restrictions:
```
Comment (segalziv) [inline scroll/readme.md:25]: currently we only reference the ES search API, but this readme need to point to the scroll API - as this is a scroll API and not just search. Maybe need to write a word or two on the paging mechanism or just link to ES one

File: scroll/readme.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+* You can not use the `max_determinized_states` element inside the `query` element
+
+### Response
+
+----------------------
+| Status code | Body    | Meaning |
+| ----------- | ------- | ---- | 
+| 200         | The result of the query, as documented in Elasticsearch Search API (link above) | Query succeeded |
+| 400         | "Found JSON elements which are not allowed [...]" | See Limits above for list of allowed elements in Query |
+| 400         | "[...] query_string.allow_leading_wildcard not allowed to be true" | |
```
Comment (segalziv) [inline scroll/readme.md:44]: I don't understand the `[...]` part. I would drop this completely

File: scroll/readme.md (hunk 1 above)
Comment (shlomialfasi) [reply scroll/readme.md:11]: changed

File: scroll/readme.md (hunk 4 above)
Comment (shlomialfasi) [reply scroll/readme.md:25]: i changed the url to point to scroll

File: scroll/readme.md (hunk 3 above)
Comment (shlomialfasi) [reply scroll/readme.md:31]: removed

File: scroll/readme.md (hunk 5 above)
Comment (shlomialfasi) [reply scroll/readme.md:44]: according to my understanding this is a place for something that is a parameter

**Review by shlomialfasi — COMMENTED** [review]
i removed the file. will add it with this fixes when we publish the API

**Pull request merged**